pkgs/development/python-modules/adb-homeassistant/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, libusb1
+, rsa
+, pycryptodome
+, m2crypto
+}:
+buildPythonPackage rec {
+  pname = "adb-homeassistant";
+  version = "1.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "344bcaa0440501c8942fb17f27566439d1f8336c32ef1853750c995d039e58e0";
+  };
+
+  propagatedBuildInputs = [
+    libusb1
+    rsa
+    pycryptodome
+  ];
+
+
+  checkInputs = [ m2crypto ];
+  # disabled as the only test implemented requires m2crypto, resulting in
+  #   undefined symbol: PyFile_Check
```

Comment:
Where did you get this info from? Anyway, tests aren't shipped on PyPI so I'd state that as a reason.

Reply:
ah, actually m2crypto for `python3.6` fails on the stable branch (the one i tested): http://paste.krebsco.de/4Fq4VkC2/+inline
when i try to build on the current unstable channel i get another error, this time however for the package itself: http://paste.krebsco.de/pG6NNqtH/+inline `AttributeError: type object 'Callable' has no attribute '_abc_registry'` - the error persists even when i use the github sources

Reply:
I've changed the package to load the github sources and essentially run the travis-ci commands. That means tests are now enabled and working